assets/prefabs/lava.prefab

```diff
@@ -0,0 +1,5 @@
+{
+ "DamagingBlock": {
+        "blockDamage": 10
+        }
+}
```

Comment:
CoreBlocks does not depend on DamagingBlocks and thus does not know about this component. It should be added by using a delta prefab in the DamagingBlocks module itself.

Similar as we do for water, use an empty prefab for lava: https://github.com/Terasology/CoreBlocks/blob/develop/assets/prefabs/water.prefab

```suggestion
{
}

```